assets/src/containers/AssignmentPlanning.js

```diff
@@ -88,7 +97,7 @@ function AssignmentPlanning (props) {
                   <MenuItem value={75}>75%</MenuItem>
                 </Select>
               </FormControl>
-              {loaded ? AssignmentTable(assignmentData.plan) : <Spinner />}
+              { tableBuilder(loaded, assignmentData) }
```

Comment:
Just curious, why provided `tableBuilder` with the `loaded` variable? 

If we make it `{loaded ? tableBuilder(assignmentData.plan) : <Spinner />}`? That way, `tableBuilder` is only called when the data is loaded. As written, `tableBuilder` is called while data is loading. Not a huge deal though, just a thought!

Reply:
Sure, done.

Reply:
Perfect, looks good to me!